src/Files.App/Services/ResourcesService.cs

```diff
@@ -45,7 +47,7 @@ public void SetAppThemeFontFamily(string contentControlThemeFontFamily)
 		/// <inheritdoc/>
 		public void ApplyResources()
 		{
-			ThemeHelper.ApplyResources();
+			AppThemeModeService.RefreshAppThemeMode();
```

Comment:
Is there a reason the method was renamed?

Reply:
Yes, this method does toggle themes between dark and light and then change to the right one. It means to refresh. It's never been apply 'resource'.

Reply:
It's not really refreshing anything, it's a workaround to force WinUI to apply the updated resource (see https://github.com/microsoft/microsoft-ui-xaml/issues/4651). `ApplyTheme` is fine with me but I prefer resources since its a little more generic.

Reply:
> any bugs
I knew that...

Is it ok if I add xml comments that mention this issue since method should not exist at the first place?

Reply:
Definitely, I think the interface would be the best location. In the meantime, please revert to `ApplyResources` or `ApplyTheme`.

Reply:
What about this one?

```cs
/// Summary:
/// Forces to load the current application theme mode.
/// Remarks:
/// This is a workaround for a WinUI bug #000.
public void ForceToLoadAppThemeMode();
```

Reply:
It's not specific to themes, we used to use this for compact spacing as well.

Reply:
I mean for IAppThemeModeService.ForceToLoadAppThemeMode();
For ResourcesService, it's good as is(ApplyResources).

Reply:
I prefer keeping the original naming

Reply:
![image](https://github.com/files-community/Files/assets/62196528/63d74637-b0ac-4f67-9a8c-afea7af9ce3f)